lib/workers/repository/updates/generate.ts

```diff
@@ -305,6 +305,14 @@ export function generateBranchConfig(
   config.dependencyDashboardPrApproval = config.upgrades.some(
     (upgrade) => upgrade.prCreation === 'approval'
   );
+  config.prBodyColumns = [
+    ...new Set(
+      config.upgrades.reduce(
+        (existing, upgrade) => existing.concat(upgrade.prBodyColumns),
+        []
+      )
+    ),
+  ];
```

Comment:
```suggestion
  ].filter(is.truthy);
```

Just in case there's a `null` or `undefined` crept in there